weave/tests/test_hypothesis.py

```diff
@@ -238,29 +260,27 @@ def test_to_compare_safe(l1):
         )
 
 
+@given(l=st.lists(obj_strategy()))
+@settings(max_examples=EXAMPLES_PER_TEST, deadline=None)
+# @example(l=[None, {"foo": 0}, 0, {"foo": ""}])
+@example(l=[None, {"foo": 15}, {"foo": "a"}])
+def test_saveload(l, weave_no_cache):
+    a = ops_arrow.to_arrow(l)
+    a_ref = storage.save(a)
+    a_again = storage.get(str(a_ref))
+    assert a.object_type == a_again.object_type
+    assert a._arrow_data == a_again._arrow_data
+
+
 @given(l1=st.lists(obj_strategy()), l2=st.lists(obj_strategy()))
-@settings(max_examples=EXAMPLES_PER_TEST)
+@settings(max_examples=EXAMPLES_PER_TEST, deadline=None)
 def test_concat(l1, l2, weave_no_cache):
     a1 = ops_arrow.to_arrow(l1)
     l1_result = a1.to_pylist_tagged()
     compare_py(l1_result, l1)
     a1 = ops_arrow.to_arrow(l1)
-
-    # There are bugs here that hypothesis finds! Commenting out for now.
-    # a1_ref = storage.save(a1)
-    # a1_again = storage.get(str(a1_ref))
-    # assert a1.object_type == a1_again.object_type
-    # assert a1._arrow_data == a1_again._arrow_data
-    # a1 = a1_again
-
```

Comment:
does this PR not fix these? should we re-enable?

Reply:
Because I moved this out into a separate test.